android/lib/src/main/java/com/ern/api/impl/core/LaunchConfig.java

```diff
@@ -9,6 +9,8 @@
 import androidx.fragment.app.Fragment;
 import androidx.fragment.app.FragmentManager;
 
+import com.ern.api.impl.navigation.NavigationRouteHandler;
```

Comment:
Hmm, so this creates a dependency from `.core` to `.navigation`. I'm pretty sure we already have several other such references, and there's already a circular dependency between these two packages.
No action item right now, but we need to remember for "Version 2.0" to rethink the package/dependency structure we're using here. Java package constraints can also be enforced through unit test via tools like [JDepend](https://github.com/clarkware/jdepend). Something we can consider eventually.

Reply:
Very good find, this was an oversight. There shouldn't be this dependency.  But I agree, looking at how this is being used we may no longer need a core package as this was initially kept for backward compatibility but no one is actually using it. 

I will fix the dependency now. Thanks.